lib/fathead/mdn_css/output.txt

```diff
@@ -344,28 +389,151 @@
+<integer> && <custom-ident>?	A										<section class="prog__container"><p>Contributes the nth grid line to the grid item’s placement. If a negative integer is given, it instead counts in reverse, starting from the end edge of the explicit grid.\n If a name is given as a <custom-ident>, only lines with that name are counted. If not enough lines with that name exist, all implicit grid lines are assumed to have that name for the purpose of finding this position.\n An <integer> value of 0 is invalid.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column#Values
+<integer> && <symbol>	A										<section class="prog__container"><p>The <integer> specifies a minimum length that all counter representations must reach. The value must be non-negative. If the minimum length is not reached, the representation will be padded with the specified <symbol>.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/@counter-style/pad#Values
+<length-percentage>	A										<section class="prog__container"><p>Is the width of the gutter separating the grid columns. <percentage> values are relative to the dimension of the element.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column-gap#Values
+<length-percentage> 	A										<section class="prog__container"><p>A positive <length> or <percentage> value. When the units of <length> values are specified in em or ex, the size is defined relative to the size of the font on the parent element of the element in question. For example, 0.5em is half the font size of the parent of the current element. When the units are specified in rem, the size is defined relative to the size of the font used by the <html> (root) element.\n <percentage> values refer to the parent element's font size.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/font-size#Values
+<length>	A			css data types							<section class="prog__container"><p>Is a <length> value giving a hint of the optimal width of the column. The actual column width may be wider (to fill the available space), or narrower (only if the available space is smaller than the specified column width). The length must be strictly positive or the declaration is invalid.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/column-width#Values
+<li>	R	list-style-type										
+<line-height>	A										<section class="prog__container"><p>See the line-height CSS property.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/font#Values
+<line-width>	A										<section class="prog__container"><p>The width of the stroke.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-text-stroke-width#Values
+<link>	R	:any-link										
+<list-style-image>	A										<section class="prog__container"><p>See list-style-image</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#Values
```

Comment:
@hchienjo if the content is just going to say "See X" then I think we should just build a redirect instead of an article?